metpy/calc/thermo.py

```diff
@@ -1205,3 +1205,43 @@ def _isen_iter(iter_log_p, isentlevs_nd, ka, a, b, pok):
 
     # output values as a list
     return ret
+
+
```

Comment:
sb_cape_cin might be better, it's shorter and surface-based cape is generally shortened to SBCAPE, so I son't think it would be confusing.